src/coreclr/gc/gc.cpp

```diff
@@ -21589,6 +21629,7 @@ void gc_heap::garbage_collect (int n)
     gc_t_join.start_ts(this);
 #endif //JOIN_STATS
     clear_gen0_bricks();
+    check_gen0_bricks();
```

Comment:
I would actually put this before `clear_gen0_bricks` and check when `gen0_bricks_cleared` is TRUE. it's more useful that way.

Reply:
It won't make a difference if gen0_bricks_cleared is TRUE, because then clear_gen0_bricks won't do anything.

If gen0_bricks_cleared is FALSE, then check_gen0_bricks will check the result of clear_gen0_bricks, which arguably isn't very useful.

I'll change it so check_gen0_bricks will only check if gen0_bricks_cleared is true, and swap the order.